### ISSUE TITLE
Don't ask about signature places if requested from the app (v5.1.4)

### ISSUE THREAD

**Issue by tasagore** [LibreSign/libresign#885]
LibreSign 5.1.4, NC24

(may be the texts in the buttons are not accurate because I'm not work

1) From NC click on the LibreSign app icon
2) Click on "Choose from Files"
3) At the right introduce on email and click in Add
4) At the right click on "Submit Request"

After that the user receives a email and it can be signed but after step 4 the app should ask before if we want to setup the visible signatures, like it does if we request the signature from the details option (although there is another bug the expected behavior should be like in https://github.com/LibreSign/libresign/issues/886#issue-1334523584).

**Comment by vitormattos** [comment]
After step 4 that you describe, you will need to answer the question about the type of signature flow that you want, with visible signature or not. Did you receive this message and reply?

**Comment by tasagore** [comment]
> After step 4 that you describe, you will need to answer the question about the type of signature flow that you want, with visible signature or not. Did you receive this message and reply?

No, the message does not appears, take a look to the other issue opened by me (886), it's a different way to request a signature, in that case the message appears, but not in this way, that's why I suggest it as a bug, the message should appear in any way to request.

**Comment by vitormattos** [comment]
I think that this issue is outdated and this already was solved.

Check the newest version of LibreSign. If you found anything with strange behavior, create a new issue.
